griffin-doc/measure/measure-configuration-guide.md

```diff
@@ -188,7 +188,7 @@ Above lists DQ job configure parameters.
 - **sinks**: Whitelisted sink types for this job. Note: no sinks will be used, if empty or omitted. 
 
 ### <a name="data-connector"></a>Data Connector
-- **type**: Data connector type, "AVRO", "HIVE", "TEXT-DIR" for batch mode, "KAFKA" for streaming mode.
+- **type**: Data connector type: "AVRO", "HIVE", "TEXT-DIR", "CUSTOM" for batch mode; "KAFKA", "CUSTOM" for streaming mode.
```

Comment:
do you think about 'ANY' as replacement for 'CUSTOM'

**type**: Data connector type, "AVRO", "HIVE", "TEXT-DIR" for batch mode, "KAFKA" for streaming mode, "ANY" for boths.

Reply:
ANY seems to have "auto-detect" meaning to it, while in fact it's just plugin mechanism.
What about CLASS, CLASSNAME, PLUGIN or EXTERNAL?

Reply:
I still think 'custom' better than others.

Reply:
Let stick to CUSTOM then.